traffic_ops/traffic_ops_golang/server/servers_server_capability.go

```diff
@@ -443,8 +444,8 @@ func getDSTenantIDsByIDs(tx *sqlx.Tx, dsIDs []int64) ([]DSTenant, error) {
 	return dsTenantIDs, nil
 }
 
-// AssignMultipleServerCapabilities helps assign multiple server capabilities to a given server.
-func AssignMultipleServerCapabilities(w http.ResponseWriter, r *http.Request) {
+// AssignMultipleServersCapabilities assigns multiple servers to a capability or multiple server capabilities to a server
+func AssignMultipleServersCapabilities(w http.ResponseWriter, r *http.Request) {
```

Comment:
This function needs to check the locking mechanism, as followed by the other endpoints.

Reply:
It uses locking mechanism `CheckIfCurrentUserCanModifyCDN()`